querybook/server/lib/query_executor/clients/trino.py

```diff
@@ -61,12 +62,15 @@ def __init__(
         host = trino_conf.host
         port = 8080 if not trino_conf.port else trino_conf.port
 
+        auth = trino.auth.BasicAuthentication(username, password)
```

Comment:
to make the behavior consistent as before, can you only provide `auth` to connect params if both username and password are provided?

Reply:
Ack